imap_processing/glows/glows_cdf_attrs.py

```diff
@@ -0,0 +1,190 @@
+"""Shared attribute values for GLOWS CDF files."""
+
+from imap_processing.cdf.defaults import GlobalConstants
+from imap_processing.cdf.global_attrs import (
+    AttrBase,
+    GlobalDataLevelAttrs,
+    GlobalInstrumentAttrs,
+    ScienceAttrs,
+)
+from imap_processing.glows import __version__
+
+text = (
+    "The GLObal Solar Wind Structure (GLOWS) is a non-imaging single-pixel Lyman-alpha "
+    "photometer that will be used to observe the sky distribution of the helioglow to "
+    "better understand the evolution of the solar wind structure. "
+    "The Lyman-alpha photon counts from these observations can be used to build a more "
+    "comprehensive picture of the solar wind structure and how it changes through the "
+    "solar cycles. "
+    "GLOWS design and assembly is led by the Space Research Center, Warsaw, Poland "
+    "(CBK PAN). See https://imap.princeton.edu/instruments/glows for more details."
+)
+
+glows_base = GlobalInstrumentAttrs(
+    __version__,
+    "GLOWS>GLObal Solar Wind Structure",
+    text,
+    "Photons (space)",  # TODO: is this accurate?
+)
+
+# TODO: Do we want a data format version as well as a software version?
+
+glows_l1a_hist_attrs = GlobalDataLevelAttrs(
+    "L1A_hist>Level-1A histogram",
+    logical_source="imap_glows_l1a_hist",
+    logical_source_desc="IMAP Mission GLOWS Histogram Level-1A Data.",
+    instrument_base=glows_base,
+)
+
+glows_l1a_de_attrs = GlobalDataLevelAttrs(
+    "L1A_de>Level-1A direct event",
+    logical_source="imap_glows_l1a_de",
+    logical_source_desc="IMAP Mission GLOWS Direct Event Level-1A Data.",
+    instrument_base=glows_base,
+)
+
+bins_attrs = AttrBase(
+    validmin=0,
+    validmax=70,  # doc says maximum count per bin is 66.7
+    catdesc="Counts of direct events for photon impacts.",
+    fieldname="Counts of direct events.",
+    format="F7.4",  # Float with 4 digits
```

Comment:
@bryan-harter Is this correct if float data is only 2 digits followed by 4 digits after the decimal?
```suggestion
    format="F2.4",  # Float with 4 digits
```